CHANGELOG.md

```diff
@@ -11,6 +11,13 @@
   returned first and in the same order in the list of results.
   ([PR #707](https://github.com/jasmin-lang/jasmin/pull/707)).
 
+- The (speculative) constant-time checker (`jazzct`) can optionally
+  (when the `-doit` flag is used) check that secrets are only used with
+  guaranteed constant time instructions (DOIT for Intel, DIT for ARM). This option
+  only makes sense when done after the lowering compiler pass, which can be ensured
+  with the new `-after` option.
```

Comment:
for -doit, did we have a default pass ?

Reply:
What do you mean by a default pass? There are two ways I guess:
 - Passing the `-doit` option also sets the compiler pass to be after inline propagation. If someone also uses the `-after` option they get an error? Or maybe they get an error only if the pass is before lowering?
 - Passing the `-doit` option does nothing to the compiler pass set, it needs to be set using the `-after` option and that will check that the set pass is after lowering. This is currently implemented (and I like it more, but I am open to suggestions).

Reply:
I was more in favor of the first one, with
the solution "Or maybe they get an error only if the pass is before lowering?". I think the user don't need 
to know the name of the pass.

Reply:
Ok, makes sense.